<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textInformation</class>
 <widget class="QDialog" name="textInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>265</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>textInformation</string>
  </property>
  <widget class="QLineEdit" name="pathLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>版本信息：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fileNameLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="versionLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileNameLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>新建储存文件</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
